multiversx_sdk/controllers/delegation_controller.py

```diff
@@ -0,0 +1,341 @@
+from typing import Any, List, Sequence
+
+from multiversx_sdk.controllers.interfaces import IAccount
+from multiversx_sdk.controllers.network_provider_wrapper import ProviderWrapper
+from multiversx_sdk.converters.transactions_converter import \
+    TransactionsConverter
+from multiversx_sdk.core.interfaces import IAddress, IValidatorPublicKey
+from multiversx_sdk.core.transaction import Transaction
```

Comment:
on imports I would've group imports based on their namespace, for example
from multiversx_sdk.core.transactions_outcome_parsers import DelegationTransactionsOutcomeParser, CreateNewDelegationContractOutcome

Reply:
Did some of them, will probably do the same in all the files in the future PRs.